### ISSUE TITLE
Rename turbulent_kinetic_energy and change units

### ISSUE THREAD

**Issue by nusbaume** [ESCOMP/CCPPStandardNames#70]
Currently the repo contains the standard name `turbulent_kinetic_energy` with units of `J`.  However, in atmospheric science the default units for TKE is `J/kg`, which are the units for the variable in both SIMA and the UFS (although the diagnostic output in the UFS has it mislabeled as just `J`).

Thus in order to clarify the units, and to better match the CF-conventions, it was decided to change the standard name in this repo to be `specific_turbulent_kinetic_energy`, and to change the units to `J/kg`.

**Comment by dudhia** [comment]
I don't think the word "specific" is needed. There are many energies like
CAPE and moist static energy that are conventionally J/kg.
For sure, J is not correct.
Jimy

**Comment by nusbaume** [comment]
@dudhia We discussed this at the CCPP framework meeting and landed on the word "specific" for two reasons:

1.  It matches what is listed in the CF-conventions, as there is no `turbulent_kinetic_energy`, only `specific_turbuent_kinetic_energy`, and the conventions make a point that the word `specific` means "per unit mass".

2.  We generally wanted to lean towards being more verbose in order to remove any chance of ambiguity.  Since technically the SI unit for energy is `J`, we thought adding `specific_` would make it clear to non-atmospheric scientists that the units should be `J kg-1`.

You do make a really good point though that other "energy" quantities are also per unit mass, and ideally it would be good to have the names of these quantities be consistent (e.g. they either all have `specific_` or none of them do).  I'll make it an agenda item for the next framework/standard names meeting to discuss it some more (although if people watching this repo have opinions then feel free to respond here on this issue as well).

**Comment by mkavulich** [comment]
@dudhia I agree that terms for which there is a standard use that implies specific quantities (such as CAPE) this renaming is not needed. This discussion is specifically (heh) about TKE. Doing some literature review (AKA googling "turbulent kinetic energy units") it seems like the standard usage implies "specific" units (energy per unit mass), so maybe this renaming is not necessary; just re-defining the units to J kg^-1 would be sufficient.

**Comment by nusbaume** [comment]
After discussing this at the CCPP-framework, we decided to keep the name as `turbulent_kinetic_energy`, but to change the units to `J kg-1`.  I'll go ahead and remove the "name changing" label to make it clear that the standard name itself will remain the same.
